install_test.ps1

```diff
@@ -17,29 +17,27 @@ if (!(Get-Module PSScriptAnalyzer -ListAvailable)) {
 # Lint.
 Invoke-ScriptAnalyzer *.ps1 -EnableExit -Exclude PSAvoidAssignmentToAutomaticVariable
 
-$BinDir = if ($IsWindows) {
-  "$Home\.deno\bin"
+# Test that we can install the latest version at the default location.
+$env:DENO_INSTALL = ""
+.\install.ps1
+if ($IsWindows) {
+  ~\.deno\bin\deno.exe --version
 } else {
-  "$Home/.deno/bin"
+  ~/.local/bin/deno --version
 }
 
-# Test we can install a specific version.
-Remove-Item $BinDir -Recurse -Force -ErrorAction SilentlyContinue
+# Test that we can install a specific version at a custom location.
+$env:DENO_INSTALL = if ($IsWindows) {
+  "$Home\deno-0.13.0"
+} else {
+  "$Home/deno-0.13.0"
+}
 .\install.ps1 v0.13.0
 $DenoVersion = if ($IsWindows) {
-  deno --version
+  ~\deno-0.13.0\bin\deno.exe --version
 } else {
-  ~/.deno/bin/deno --version
+  ~/deno-0.13.0/bin/deno --version
 }
 if (!($DenoVersion -like '*0.13.0*')) {
   throw $DenoVersion
 }
-
-# Test we can install the latest version.
```

Comment:
why remove this?

Reply:
It's just moved up to match the shell version. Sorry for the confusion.

Reply:
I think you meant to click on the `Remove-Item`. I don't know why I removed those! Restored.